smart-contracts/truffle-config.js

```diff
@@ -56,12 +62,10 @@ module.exports = {
           enabled: true,
           runs: 200,
         },
-        evmVersion: 'petersburg',
```

Comment:
Why remove this?  It's a compiler setting which enables the most recent bytecode optimizations and new features like CREATE2 (which we don't use yet)

Reply:
Ohhhh, not sure but I guess it's because I did not rebase.